invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/Profile/ProfileContainer.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { Container } from 'semantic-ui-react';
+import {
+  PatronCurrentLoans,
+  PatronMetadata,
+  PatronPastLoans,
+  PatronPendingLoans,
+} from './components';
+import { sessionManager } from '../../../authentication/services';
+
+export default class ProfileContainer extends Component {
+  render() {
+    const currentUser = sessionManager.user;
+    return (
+      <Container>
+        <PatronMetadata username={currentUser.username} />
+        <PatronPendingLoans patronPid={currentUser.id} />
+        <PatronCurrentLoans patronPid={currentUser.id} />
```

Comment:
The prop for `PatornCurrentLoans` is called `patron` but should probably be renamed like the others.